framework/components/ecl-carousels/ecl-carousels.config.js

```diff
@@ -0,0 +1,95 @@
+module.exports = {
+  title: 'Carousels',
+  label: 'Carousels',
+  status: 'ready',
+  tags: ['molecule'],
+  context: {
+    _demo: {
+      scripts: `
+        document.addEventListener('DOMContentLoaded', function () {
+            ECL.carousels();
+        });
+      `,
+    },
+    extra_attributes: [
+      {
+        name: 'aria-labelledby',
```

Comment:
I left them out of template mainly because I imagine that some carousels will not have the headings. And the `aria-labelledby` will be wrongly used. So, having them both optionally avoids the possibility of wrong use because of a missing variable.